robottelo/test.py

```diff
@@ -73,6 +73,7 @@ def setUpClass(cls):
 class APITestCase(TestCase):
     """Test case for API tests."""
     _multiprocess_can_split_ = True
+    longMessage = True
```

Comment:
This should be already inherited, look at line 70.

The `longMessage` is something that the entire framework would benefit from. If that is not working (it was last time I checked) please could you fix that instead?